parm/config/config.resources

```diff
@@ -137,9 +137,9 @@ elif [ $step = "anal" ]; then
       export npe_anal=84
       export npe_anal_gfs=84
     fi
-    if [[ "$machine" = "WCOSS_DELL_P3" ]]; then export nth_anal=7; fi
```

Comment:
Why is this line that sets `nth_anal` for the Dells removed?

Reply:
It is because @aerorahul requested " I suggest remove the QUEUE logic and replace 4 with 5 on Line 141." above.

Reply:
`if [[ "$machine" = "WCOSS_DELL_P3" ]]; then export nth_anal=7; fi`
The line you removed is not QUEUE logic, it's machine logic. Please put it back.

Reply:
To my understand, @aerorahul requested this change. "replace 4 with 5 on Line 141" is in place. Please get approve from @aerorahul if you want to back out this change.

Reply:
Based on /gpfs/dell2/emc/modeling/noscrub/emc.gloparadev/para_fv3gfs/v16iasicris2/config.resources. Line 125 was removed from DA team. export nth_anal=4 is used in v16iasicris2 final configuration. That is the version I purposed originally. Here, @aerorahul requested to use nth_anal=5 instead. Thus, the final became nth_anal=5.
Logic nth_anal inside WCOSS_DELL_P3 if statement is equal to the setting before on line 130. Therefore, it is not need to have extra assignment.

Reply:
Lin, what machines was this `nth_anal` change tested on? If it was tested on every supported platform then I am ok with removing the machine logic and setting `nth_anal` to the same value everywhere (whatever value the DA team prefers). If this was only tested on WCOSS-Dell phase 3 then please use what @aerorahul suggested:
```
if [[ "$machine" = "WCOSS_DELL_P3" ]]; then export nth_anal=4; fi
```